docs/Using Fleet/enroll-hosts.md

```diff
@@ -114,9 +115,28 @@ To generate an osquery installer for a team:
 3. Next, select **Add hosts** and copy the `fleetctl package` command for the platform (macOS, Windows, Linux) of the hosts you'd like to add to a team in Fleet.
 4. Run the copied `fleetctl package` command and [distribute your installer](#adding-multiple-hosts) to add your hosts to a team in Fleet.
 
+### Generating Windows installers using local WiX toolset v3 binaries on Windows
+
+When creating a Fleetd installer for Windows hosts (**.msi**) on a Windows machine, you can tell `fleetctl package` to
+use local installations of the 3 WiX v3 binaries used by this command (`heat.exe`, `candle.exe`, and
+`light.exe`) instead of those in a pre-configured container, which is the default behavior. To do
+so:
+  0. If you need to install the WiX v3 binaries, you can download them
+     [here](https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip).
+     Unzip the downloaded file.
```

Comment:
Built this page locally to preview, and looks like "0." doesn't turn into a list item:
![Screenshot 2023-09-22 at 1 50 37 PM](https://github.com/fleetdm/fleet/assets/3065949/ec291904-ec57-46cf-bb9a-3e2418a8bfc6)
I'd suggest just making this step 1. If they've already done it, they'll know to skip this step 😄 
```suggestion
  1. Install the WiX v3 binaries. To install, you can download them
     [here](https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip), then unzip the downloaded file.
```